docs/section19/19-6.md

```diff
@@ -0,0 +1,35 @@
+# Section 19 - Rule 19-6      
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True      
+**Rule ID:** 19-6         
+**Rule Description:** Unmet load hours for the baseline design shall not exceed 300 (of the 8760 hours simulated).      
+**Rule Assertion:** Options are Pass/Fail     
+**Appendix G Section:** G3.1.2.3        
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each Baseline Design 0 Degree Rotation RulesetModelInstance  (should only be one per project)
+
+**Applicability Checks:**  
+
+1. Applies only to the baseline design (no rotation) rule set model instance.  
+
+**Function Calls:**  None  
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- For each RulesetModelInstance in ASHRAE229.ruleset_model_instance : `for RulesetModelInstance in ASHRAE229.ruleset_model_instance:`  
+    - Check if the ruleset model instance is the no rotation baseline model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "BASELINE_0":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.OutputInstance.unmet_load_hours_cooling`  
+
+        **Rule Assertion:** 
+        - Case 1: If the heating + cooling unmet load hours is 300 or less then pass: `if unmet_load_hours_heating + unmet_load_hours_cooling <= 300: outcome = "PASS"`   
+        - Case 2: Else, fail: `Else: outcome = "Fail"`  
+
+**Notes/Questions:**  
+1. Does this apply at the ruleset model instance level as I created it? I wasn't sure how to reference the output schema. I took a shot at it here. 
+2. Do we check this for each rotation of the baseline model or just 0 degrees (i.e., as-designed orientation)? When I do reviews I only check the design orientation but I want to make sure that is correct.  
```

Comment:
It seems it shall be depends on whether the `unmet_load_hours` is one metric of `baseline building performance`?
We may need to add a check for rotation exceptions if averaging the hours.

Reply:
Per call, MK agree with JG that okay to just check as-designed rotation.